bids/analysis/transformations/compute.py

```diff
@@ -33,8 +34,13 @@ def _transform(self, var, model='spm', derivative=False, dispersion=False,
 
         model = model.lower()
 
+        post_resample = False
         if isinstance(var, SparseRunVariable):
             sr = self.collection.sampling_rate
+            min_sampling_rate = 1 / var.duration.min()
```

Comment:
Can duration end up being an int? In python 2 then end up with 0 . I would have been paranoid and used 1.0 not 1 just in case